src/components/svgs/RedeemBtcLogo.js

```diff
@@ -0,0 +1,37 @@
+import React from "react"
+import PropTypes from "prop-types"
+
+const RedeemBtcLogo = ({ width = 104, height = 111, fill = "#080503" }) => {
```

Comment:
Wonder if this can just be `RedeemIcon` or similar. It's not really a logo per se… Or is it…